2-mandatory/6-budgets.js

```diff
@@ -10,302 +10,317 @@ getBudgets([
 Should give return the answer of 62600.
 **/
 
-function getBudgets(peopleArray) {
-}
+const getBudgets = (peopleArray) => {
+  let budget = 0;
+  for (people of peopleArray) {
+    budget += people.budget;
+  }
+  return budget;
```

Comment:
As always great code! But since we talked about using `reduce`, this is a perfect example! You can do:
`const getBudgets = (peopleArray) => peopleArray.reduce((a, b) => a +  b.budget, 0)`